apps/communications/contacts/test/marionette/list_test.js

```diff
@@ -51,6 +51,68 @@ marionette('Contacts > List', function() {
     });
   });
 
+  suite('List Order > ', function() {
+    var data = [['GG', 'E'], ['AA', 'Z'],
+      ['XX', 'C'], ['CC', 'X'], ['EE', 'G'], ['FF', 'F'], ['HH', 'D'],
+      ['BB', 'Y'], ['YY', 'B'], ['ZZ', 'A'], ['DD', 'H']];
+    setup(function() {
+      data.forEach(contactData => {
+        subject.addContact({
+          givenName: contactData[0],
+          familyName: contactData[1]
+        });
+      });
+    });
+
+    function getNamesInList() {
+      return client.executeScript(function() {
+        var names = [];
+        var namesNodes = document.querySelectorAll('li:not([data-group="ice"]).contact-item p');
+        namesNodes = Array.prototype.slice.call(namesNodes);
+        namesNodes.forEach((name) => {
+          names.push(name.textContent)
+        });
+
+        return names;
+      });
+    }
+
+    function givenNameOrder(a, b) {
+      return a[0].localeCompare(b[0]);
+    }
+
+    function familyNameOrder(a, b) {
+      return a[1].localeCompare(b[1]);
+    }
+
+    test('Order by given name > ', function() {
```

Comment:
This test name is a bit ambiguous: do we test the case where the user ask to order by given name, or do we test the default order?

Based on the current implementation, I'd rebrand it to something like: 'Given name is the default order'